config/application.rb

```diff
@@ -19,7 +19,7 @@
 module Mmt
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 5.2
```

Comment:
that looks like something we missed, should we request a full regression test because of this?

Reply:
Yes, I think so.

Reply:
https://apidock.com/rails/v5.2.3/Rails/Application/Configuration/load_defaults not that many changes. 5 maybe.